interfaces/Portalicious/src/app/domains/project/project.helper.ts

```diff
@@ -13,14 +16,62 @@ import {
   Project,
 } from '~/domains/project/project.model';
 
-export const ATTRIBUTE_LABELS: Record<string, string | undefined> = {
-  fspDisplayName: $localize`:@@attribute-label-fspDisplayName:FSP Display Name`,
+export const ATTRIBUTE_LABELS: Record<GenericRegistrationAttributes, string> = {
+  referenceId: $localize`:@@attribute-label-referenceId:Reference ID`,
+  phoneNumber: $localize`:@@attribute-label-phoneNumber:Phone Number`,
+  preferredLanguage: $localize`:@@attribute-label-preferredLanguage:Preferred Language`,
   paymentAmountMultiplier: $localize`:@@attribute-label-paymentAmountMultiplier:Transfer value multiplier`,
+  programFinancialServiceProviderConfigurationName: $localize`:@@attribute-label-programFinancialServiceProviderConfigurationName:FSP Configuration Name`,
+  programFinancialServiceProviderConfigurationLabel: $localize`:@@attribute-label-programFinancialServiceProviderConfigurationLabel:FSP Display Name`,
   maxPayments: $localize`:@@attribute-label-maxPayments:Max. payments`,
+  paymentCount: $localize`:@@attribute-label-paymentCount:Payment Count`,
   paymentCountRemaining: $localize`:@@attribute-label-paymentCountRemaining:Remaining payments`,
+  scope: $localize`:@@attribute-label-scope:Scope`,
+  status: $localize`:@@attribute-label-status:Status`,
+  registrationProgramId: $localize`:@@attribute-label-registrationProgramId:Registration ID`,
+  registrationCreatedDate: $localize`:@@attribute-label-registrationCreatedDate:Registration Created Date`,
+  inclusionScore: $localize`:@@attribute-label-inclusionScore:Inclusion Score`,
+};
+
+export const getGenericAttributeDataListItem = (
```

Comment:
We (me and @aberonni ) had an open ended discussion if the logic of this function should live should live in the backend or in the frontend. 

At first I came to a conclusion to add it to the backend in [/api/programs/{programId}/attributes](http://localhost:3000/docs/#/programs/ProgramController_getAttributes) under another flag like `includeTemplateDefaultAttributes` and map the typings there. However I thought later it does not make sense to categorize the `GenericRegistrationAttributes` using the `RegistrationAttributeTypes` as this enum is meant for categorizing `ProgramRegistrationAttributeEntities`. For example `programFinancialServiceProviderConfigurationLabel` does not fall in any category and preferredLanguage and programFinancialServiceProviderConfigurationName are sort of dropdown but the do not work the same as the 'normal' dropdown attributes.

The major reason of setting the RegistrationAttributeTypes in the backend is so that it could be used in `projectAttributeToDataListItem` to map it to the type of a `DataListItem (a frontend) conecpt. I therefore decided to create this function `getGenericAttributeDataListItem` in the frontend directly.

For the upcoming work of edit attribute data on the profile page I think that a lot of the editable GenericRegistrationAttributes are sort of `exceptions` that need their own logic (like programFspConfig or the preferredLanguage). So the may also not need the RegistrationAttributeTypes as parameter of they should be edited in the frontend